python/ray/services.py

```diff
@@ -410,11 +418,23 @@ def start_ray_processes(address_info=None,
   # warning messages when it starts up. Instead of suppressing the output, we
   # should address the warnings.
   redis_address = address_info.get("redis_address")
-  if redis_address is None:
-    redis_address = start_redis(node_ip_address, cleanup=cleanup,
-                                redirect_output=redirect_output)
-    address_info["redis_address"] = redis_address
-    time.sleep(0.1)
+  if include_redis:
+    if redis_address is None:
+      redis_address = start_redis(node_ip_address, cleanup=cleanup,
+                                  redirect_output=redirect_output)
+      address_info["redis_address"] = redis_address
+      time.sleep(0.1)
+    else:
+      redis_host, redis_port = redis_address.split(":")
```

Comment:
the current terminology is
blah_ip_address -> ip_address of blah without port,
blah_port -> port of blah
blah_address -> ip address and port of blah

I like blah_host better instead of blah_ip_address, but we should make it the same everywhere